simple_network_sim/loaders.py

```diff
@@ -250,6 +246,25 @@ def readTrials(df: pd.DataFrame) -> int:
         return trials
 
 
+def readStartEndDate(df: pd.DataFrame) -> Tuple[datetime.date, datetime.date]:
+    """
+    Transforms the dataframe from the data API into a bool int inside the model
+    :param df: The dataframe containing the starting date of the model
+    :return: The starting date of the model
+    """
+    if len(df) != 2:
+        raise ValueError("DataFrame must be of size 2")
+    if not all(df.columns == ["Parameter", "Value"]):
+        raise ValueError("There must be 2 columns names 'Parameter' and 'Value'")
+    if ("start_date" not in list(df.Parameter)) or ("end_date" not in list(df.Parameter)):
+        raise ValueError("Both a start and end date must be provided")
+
+    start_date = datetime.datetime.strptime(df.at[0, "Value"], '%m/%d/%Y').date()
```

Comment:
This way no risk to have any time error added. The format of only providing a date should be strictly respected.